ecc/Makefile

```diff
@@ -15,19 +17,28 @@ VSCC_OUT = ecc-vscc.so
 
 DOCKER_IMAGE ?= $$(docker images | grep -- ${NET_ID}-${PEER_ID}-$(CC_NAME)- | awk '{print $$1;}')
 DOCKER_CONTAINER_ID?=$$(docker ps | grep -- ${NET_ID}-${PEER_ID}-$(CC_NAME)- | awk '{print $$1;}')
+DOCKER_BOILERPLATE_ECC_IMAGE ?= $(PROJECT_NAME)/boilerplate-ecc
 
-
-all: build test
-build: ecc vscc-plugin
+all: build
+# test target disable because it requires revision
+#all: build test
+build: docker-boilerplate-ecc vscc-plugin
 
 ECC_ENCLAVE_BUILD = ../ecc_enclave/_build
 ERR_MSG = "ecc_enclave build does not exist!"
 
-ecc_enclave:
+ecc_enclave: sym-links
```

Comment:
I guess all that sym-linking is also due to not-yet removing/outsourcing MRENCLAVE? If so, maybe also add comment as for docker files?

Reply:
done